sepsis-quiz-js/src/sepsisQuiz.js

```diff
@@ -97,11 +105,6 @@ class SepsisQuiz {
       answer = answer ? allOfTheAbove : noneOfTheAbove
       choices = SepsisQuiz.shuffleArray(wrongAnswers).concat(answer)
     }
-    // when the one of the incorrect answers is "all of the  above" or "none of the above"
```

Comment:
Yeah this code just doesn't work (it removes the wrong answer but never adds it to the end of the choices), and it broke none of the above.